### ISSUE TITLE
Mining: pay forward coinbases in the absence of a sortition

### ISSUE THREAD

**Issue by jcnelson** [blockstack/stacks-blockchain#2004]
If a Bitcoin block has no sortition (at height _N_), then any Stacks block mined in a subsequent sortition that builds off of any Stacks chain tip that existed at the penultimate sortition (at height _N-1_) may claim its coinbase.  This encourages miners to keep mining even if Bitcoin fees are high.

**Comment by kantai** [comment]
The initial mining bonus is a special case of this, with some additional features (the reward isn't given to one block, but spread out):

```
The STX block rewards that are “accumulated” prior to the actual launch of
Stacks 2.0 (the launch block) will be added to some initial number of blocks
(e.g., 10,000 blocks or roughly 10 weeks), in order to ensure added incentive
for the start of mining initially.
```

**Comment by muneeb-ali** [comment]
Linking the forum post here where the supply + bonus was discussed: https://forum.stacks.org/t/pox-consensus-and-stx-future-supply/11232?u=muneeb

**Comment by kantai** [comment]
Closed by #2101
